Sources/XiEditor/Document.swift

```diff
@@ -64,12 +64,12 @@ class Document: NSDocument {
         // save this before instantiating because instantiating overwrites with the default position
         let newFrame = frameForNewWindow()
         var windowController: NSWindowController!
-        let storyboard = NSStoryboard(name: NSStoryboard.Name(rawValue: "Main"), bundle: nil)
+        let storyboard = NSStoryboard(name: "Main", bundle: nil)
         windowController = (storyboard.instantiateController(
-            withIdentifier: NSStoryboard.SceneIdentifier(rawValue: "Document Window Controller")) as! NSWindowController)
```

Comment:
there are a bunch of changes like this that seem like they're maybe related to a swift version change? Or is this something else?

Reply:
Those are all related to the version change. With SwiftPM all targets must be the same version, so I figured it would be better to update the main target than create a new target in an old version. I'll change it back if you want.

Reply:
Just for the sake of easier review I think it would be good to make these changes separately. I don't think it's a problem if we have a new target in an old version, for now? You could also open a separate PR updating to 4.2 and we can discuss there, it may well be worth it I haven't been following swift evolution that closely.

Reply:
Ok, reverting it now.

Reply:
It's reverted!